docusaurus.config.ts

```diff
@@ -165,6 +165,56 @@ const config: Config = {
       additionalLanguages: ['powershell'],
     },
   } satisfies Preset.ThemeConfig,
+
+  plugins: [
+    [
+      'docusaurus-plugin-llms',
+      {
+        // Options here
+        generateLLMsTxt: true,
+        generateLLMsFullTxt: true,
+        docsDir: 'docs',
+        ignoreFiles: ['advanced/*', 'private/*'],
```

Comment:
Left over from template? Paths don't exist in project so maybe remove to avoid a potential future issue?